Packages/vcs/Lib/editors/label.py

```diff
@@ -152,6 +172,9 @@ def delete(self):
     def detach(self):
         super(LabelEditor, self).detach()
         self.toolbar.detach()
+        tprop = self.actor.GetTextProperty()
+        tprop.SetBackgroundColor(*self.real_bg)
+        tprop.SetBackgroundOpacity(self.real_bg_opacity)
```

Comment:
Reset the background color and opacity to whatever they were before we began working on this label.